plugins/IntranetMeasurable/lang/en.json

```diff
@@ -0,0 +1,7 @@
+{
+    "IntranetMeasurable": {
+        "Intranet": "Intranet",
+        "Intranets": "Intranets",
+        "IntranetDescription": "An intranet measurable is just like a website but hosted on an internal network."
```

Comment:
Since an intranet is the actual internal network, do you think "Intranet Website" might be a better term to use?